include/wimlib/decompress_common.h

```diff
@@ -400,17 +400,22 @@ read_huffsym(struct input_bitstream *is, const u16 decode_table[],
  * structure, then the outer structure must be allocated on a
  * DECODE_TABLE_ALIGNMENT-byte aligned boundary as well.
  */
+#ifdef _MSC_VER
+#pragma pack(push, 16)
```

Comment:
Maybe better to define PRAGMA_BEGIN_PACKED(x) / PRAGMA_END_PACKED macros, as this could reduce all this 3 line boilerplate to a single less conspicuous line.